inf-clojure.el

```diff
@@ -205,7 +206,7 @@ process to send to, when you have more than one Clojure process around.  This
 is determined by the global variable `inf-clojure-buffer'.  Suppose you
 have three inferior Clojures running:
     Buffer              Process
-    foo                 inf-clojure
+    foo                 `inf-clojure'
```

Comment:
why did you add the quotes to this line but not the line below it? Adding these formatting changes into this PR also clouds where the work is.

Reply:
One of the requirements for PRs is to run `M-x checkdoc`. It forced me to add quotes here and there and fix some other things. I can revert it, but then there will be warnings.